src/extension.ts

```diff
@@ -19,6 +19,8 @@ import { CompileTools } from "./api/CompileTools";
 import { HelpView } from "./views/helpView";
 import { ProfilesView } from "./views/ProfilesView";
 import * as Debug from './api/debug';
+
```

Comment:
Aren't a line like the following missing here?

```suggestion
import IFSBrowser from "./views/ifsBrowser.js";
```